lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -229,7 +229,12 @@ def always_ignore_tables
               nil
             end
 
-          ['schema_info', 'schema_migrations', 'ar_internal_metadata', sessions_table].compact
+          [
+            ActiveRecord::Base.schema_migrations_table_name,
+            ActiveRecord::Base.internal_metadata_table_name,
+            sessions_table,
+            'schema_info'
```

Comment:
I like 'schema_info` at the top next to the new `schema_migrations_table_name`.
```suggestion
            'schema_info',
            ActiveRecord::Base.schema_migrations_table_name,
            ActiveRecord::Base.internal_metadata_table_name,
            sessions_table
```